.pytool/Plugin/RustHostUnitTestPlugin/RustHostUnitTestPlugin.py

```diff
@@ -20,9 +21,14 @@ def RunsOnTargetList(self) -> List[str]:
         return ["NO-TARGET"]
 
     def RunBuildPlugin(self, packagename, Edk2pathObj, pkgconfig, environment, PLM, PLMHelper, tc, output_stream):
-
         ws = Edk2pathObj.WorkspacePath
         rust_ws = PLMHelper.RustWorkspace(ws)  # .pytool/Plugin/RustPackageHelper
+        
+        with_coverage = pkgconfig.get("CalculateCoverage", True)
+        
+        if platform.system() == "Windows" and platform.machine() in ["i386", "i686"]:
+            logging.debug("Coverage skipped by plugin, not supported on Windows ARM")
+            with_coverage = False
```

Comment:
Are the archs what you intended? `i386` and `i686` are x86 architecture but the message mentions Arm.

Reply:
@makubacki Thanks, not sure how I misread which machine type to look for. I've updated to look for 'aarch64'.